### PR TITLE
node: topologymgr: docs: Kubelet Topology Manager graduation to GA

### PR DESCRIPTION
**Pull request by swatisehgal**
Docs update for Kubelet Topology Manager graduation to GA.
Issue: https://github.com/kubernetes/enhancements/issues/693#issuecomment-1401799352

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/topology-manager.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/topology-manager.md:1]: You'll need to revise this text:
```markdown
### Enable the Topology Manager feature
Support for the Topology Manager requires `TopologyManager`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled.
It is enabled by default starting with Kubernetes 1.18.
```

I recommend removing it, and putting a simpler explanation just under the `feature-state` shortcode.

For example:
```markdown
If you are using a version of Kubernetes older than v1.27, switch to the documentation for that
version of Kubernetes.
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -317,6 +315,9 @@ For a reference to old feature gates that are removed, please refer to
 | `StatefulSetMinReadySeconds` | `false` | Alpha | 1.22 | 1.22 |
 | `StatefulSetMinReadySeconds` | `true` | Beta | 1.23 | 1.24 |
 | `StatefulSetMinReadySeconds` | `true` | GA | 1.25 | - |
+| `TopologyManager` | `false` | Alpha | 1.16 | 1.17 |
+| `TopologyManager` | `true` | Beta | 1.18 | 1.26 |
+| `TopologyManager` | `true` | Alpha | 1.27 | - |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:320]:
```suggestion
| `TopologyManager` | `true` | GA | 1.27 | - |
```

File: content/en/docs/tasks/administer-cluster/topology-manager.md
Comment (swatisehgal) [reply content/en/docs/tasks/administer-cluster/topology-manager.md:1]: Ack, Thanks. Updated.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -715,6 +716,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `TopologyManager`: Enable a mechanism to coordinate fine-grained hardware resource
   assignments for different components in Kubernetes. See
   [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/).
+  Topology Manager graduated to GA in 1.27 and is enabled by default. If you are using a version of
+  Kubernetes older than v1.27, switch to the documentation for that version of Kubernetes.
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:720]: This two lines can be safely dropped.

Comment (mickeyboxell) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:720]: @swatisehgal Please drop the two lines when you have a moment and we can merge this PR.

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-node-pr-reviews : PTAL for a tech sign-off.

**Comment by tengqm** [comment]
@swatisehgal Mind give this another stab?

File: content/en/docs/tasks/administer-cluster/topology-manager.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/topology-manager.md:1]: nit (doesn't block merge)

For GA, we _prefer_ to follow the style guide. For example, [headings within the text should be in sentence case](https://kubernetes.io/docs/contribute/style/style-guide/#headings).

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 2 above)
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:720]:
```suggestion
```

**Comment by swatisehgal** [comment]
@sftim @tengqm Thanks for reviewing this PR and driving it to conclusion in my absence due to being on holidays. I truly appreciate it!
I will address any pending non-blocking comments in a separate PR once we are back from Kubecon. Thanks again!

**Pull request merged**